src/nmark/def.nim

```diff
@@ -54,7 +54,7 @@ type
     tableBlock
 
   Block* = ref BlockObj
-  BlockObj = object
+  BlockObj {.acyclic.} = object
```

Comment:
A small optimization for ORC - since, as far as I'm aware, you never create cycles with Block, we can mark it as acyclic so ORC won't check it for cycles.